js/gateio.js

```diff
@@ -3506,6 +3515,19 @@ module.exports = class gateio extends Exchange {
         });
     }
 
+    parseAccount (account) {
```

Comment:
Can we avoid introducing `parseAccount` here?

Reply:
The same logic is used for both toAccount and fromAccount, so we would have to write the code twice to avoid it

Reply:
Let's clean out conflicts in this PR.